system-parachains/constants/src/lib.rs

```diff
@@ -57,3 +57,42 @@ pub const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(
 
 /// Treasury pallet id of the local chain, used to convert into AccountId
 pub const TREASURY_PALLET_ID: PalletId = PalletId(*b"py/trsry");
+
+/// Parameters supporting async backing functionality.
+///
+/// Once all system chains have migrated to the new async backing mechanism, the parameters
+/// in this namespace will replace those currently defined in `super::*`.
+pub mod fast {
+	use frame_support::weights::{constants::WEIGHT_REF_TIME_PER_SECOND, Weight};
+	pub use parachains_common::BlockNumber;
+	use sp_runtime::Perbill;
+
+	/// This determines the average expected block time that we are targeting.
+	///
+	/// Blocks will be produced at a minimum duration defined by `SLOT_DURATION`. `SLOT_DURATION` is
+	/// picked up by `pallet_timestamp` which is in turn picked up by `pallet_aura` to implement `fn
+	/// slot_duration()`.
+	///
+	/// Change this to adjust the block time.
+	pub const MILLISECS_PER_BLOCK: u64 = 6_000;
+	pub const SLOT_DURATION: u64 = MILLISECS_PER_BLOCK;
+
+	// Time is measured by number of blocks.
+	pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
+	pub const HOURS: BlockNumber = MINUTES * 60;
+	pub const DAYS: BlockNumber = HOURS * 24;
+
+	/// We assume that ~5% of the block weight is consumed by `on_initialize` handlers. This is
+	/// used to limit the maximal weight of a single extrinsic.
+	pub const AVERAGE_ON_INITIALIZE_RATIO: Perbill = Perbill::from_percent(5);
+
+	/// We allow `Normal` extrinsics to fill up the block up to 85%, the rest can be used by
+	/// Operational  extrinsics.
+	pub const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(85);
```

Comment:
Saw it in the description, but why now 85 instead of 75?

Reply:
before it was 75%, ~ 0.125 of a sec. with the same ratio, its gonna be 0.5 sec for operational, which seems to be quite high, since its allocated in every block, even if there is no operational work. With 85% we have 0.3sec which is 2 times more than previous capacity anyway.